Libplanet.Tests/Blockchain/Policies/VolatileStagePolicyTest.cs

```diff
@@ -10,5 +15,29 @@ public class VolatileStagePolicyTest : StagePolicyTest
 
         protected override IStagePolicy<DumbAction> StagePolicy =>
             _stagePolicy;
+
+        [Fact]
+        public void Lifetime()
+        {
+            TimeSpan timeBuffer = TimeSpan.FromSeconds(2.5);
+
+            Transaction<DumbAction> tx = Transaction<DumbAction>.Create(
+                0,
+                _key,
+                _fx.GenesisBlock.Hash,
+                Enumerable.Empty<DumbAction>(),
+                timestamp: DateTimeOffset.UtcNow - _stagePolicy.Lifetime + timeBuffer
+            );
+            _stagePolicy.Stage(_chain, tx);
+            Assert.True(_stagePolicy.HasStaged(_chain, tx.Id, false));
+            Assert.Equal(tx, _stagePolicy.Get(_chain, tx.Id, false));
+            Assert.Contains(tx, _stagePolicy.Iterate(_chain));
+
+            Thread.Sleep(timeBuffer * 2);
```

Comment:
Sadly, it seems not supported in some platforms. 😢 

```
Blockchain/Policies/VolatileStagePolicyTest.cs(36,26): error CS0019: Operator '*' cannot be applied to operands of type 'TimeSpan' and 'int' [/Users/runner/work/1/s/Libplanet.Tests/Libplanet.Tests.csproj]
```

Reply:
Okay, I'm going to simply call it twice instead.